### PR TITLE
Level info serialization

### PR DESCRIPTION
**Pull request by DwangDistracted**
This allows the level info files to be serialized into xml files and deserialized from xml files.

### CODE REVIEW

**Comment by DwangDistracted** [comment]
We should consider making this a Maven Project

**Comment by dershao** [comment]
Just out of curiosity, why do we need the License files?

File: src/assets/Zombie.java [hunk 1]
```diff
@@ -1,245 +1,244 @@
-package assets;
-
```
Comment (dershao) [inline src/assets/Zombie.java]: Why was this removed?

Comment (DwangDistracted) [reply src/assets/Zombie.java]: A Merge Error when I was trying to update this branch. I corrected it but the changes still show up.

Comment (DwangDistracted) [reply src/assets/Zombie.java]: Same goes for levelloader

File: .classpath [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
 	<classpathentry kind="src" path="src"/>
 	<classpathentry kind="src" path="images"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER">
+		<attributes>
+			<attribute name="module" value="true"/>
+		</attributes>
+	</classpathentry>
 	<classpathentry kind="con" path="org.eclipse.jdt.junit.JUNIT_CONTAINER/5"/>
+	<classpathentry kind="lib" path="D:/_School_/SYSC3110/L1G2_Milestone_4/3rdparty/activation-1.1.1.jar"/>
```
Comment (dershao) [inline .classpath]: This needs to be changed to a relative path or it wouldn't run except for the machine you are on.

File: src/levels/LevelLoader.java [hunk 3]
```diff
@@ -1,266 +1,338 @@
...
+	public static void reset() {
+		currentLevel = 0;
+	}
+	
+	/**
+	 * Factory Class for building LevelInfos. Contains some default values.
+	 * @author david
+	 *
+	 */
+	public class LevelFactory {
```
Comment (dershao) [inline src/levels/LevelLoader.java]: By the way, this chained method calls from this class is cool.

Comment (DwangDistracted) [reply src/levels/LevelLoader.java]: I don't like factories that spread themselves across multiple lines

**Comment by dershao** [comment]
Was implementing Serializable necessary for JaxB?
